config/routes.rb

```diff
@@ -6,11 +6,20 @@
   devise_for :users
 
   resources :users, shallow: true, except: [:index] do
-    resources :posts, except: [:index]
-    resources :articles, except: [:index]
+    resources :posts, except: [:index] do
```

Comment:
No need to use brackets if you pass only one action

Reply:
Got it, deleted brackets.